tests/quick_test.py

```diff
@@ -0,0 +1,29 @@
+import sys
+import unittest
+
+from google.appengine.ext import testbed
+from webtest import TestApp
+
+sys.path.append('./main')
+
+
+class AppTest(unittest.TestCase):
+  def setUp(self):
```

Comment:
```suggestion
  def setup(self):
```

Reply:
No -- this has to override the parent method of this name.

Reply:
Same for here I guess...